cluster/ci/config/prow/plugins.yaml

```diff
@@ -29,26 +29,48 @@ config_updater:
       name: labels
     ci-operator/config/kubernetes/descheduler/master.json:
       name: ci-operator-kubernetes-descheduler
+    ci-operator/config/kubernetes/descheduler/release-3.10.json:
+      name: ci-operator-kubernetes-descheduler
     ci-operator/config/kubernetes/metrics-server/master.json:
       name: ci-operator-kubernetes-metrics-server
+    ci-operator/config/kubernetes/metrics-server/release-3.10.json:
+      name: ci-operator-kubernetes-metrics-server
     ci-operator/config/kubernetes/coredns/master.json:
       name: ci-operator-kubernetes-coredns
+    ci-operator/config/kubernetes/coredns/release-3.10.json:
+      name: ci-operator-kubernetes-coredns
     ci-operator/config/kubernetes/autoscaler/master.json:
       name: ci-operator-kubernetes-autoscaler
+    ci-operator/config/kubernetes/autoscaler/release-3.10.json:
+      name: ci-operator-kubernetes-autoscaler
     ci-operator/config/kubernetes/cluster-capacity/master.json:
       name: ci-operator-kubernetes-cluster-capacity
+    ci-operator/config/kubernetes/cluster-capacity/release-3.10.json:
+      name: ci-operator-kubernetes-cluster-capacity
     ci-operator/config/openshift/origin/master.json:
       name: ci-operator-openshift-origin
+    ci-operator/config/openshift/origin/release-3.10.json:
+      name: ci-operator-openshift-origin
     ci-operator/config/openshift/installer/master.json:
       name: ci-operator-openshift-installer
+    ci-operator/config/openshift/installer/release-3.10.json:
+      name: ci-operator-openshift-installer
     ci-operator/config/openshift/origin-web-console-server/master.json:
       name: ci-operator-openshift-origin-web-console-server
+    ci-operator/config/openshift/origin-web-console-server/release-3.10.json:
+      name: ci-operator-openshift-origin-web-console-server
     ci-operator/config/openshift/console/master.json:
       name: ci-operator-openshift-console
+    ci-operator/config/openshift/console/release-3.10.json:
```

Comment:
I think today this means that whichever configmap gets touched will clobber the entire configmap, not just the key. need my upstream change